src/api/printable_result.py

```diff
@@ -13,6 +14,46 @@ def print(self):
         stringifier = ConsoleStringifier(c.configuration.to_stringifier_config())
         print(stringifier.result_to_str(self._result))
 
+    def get_str(self) -> str:
+        stringifier = ConsoleStringifier(c.configuration.to_stringifier_config())
+        return stringifier.create_str(self._result)
+
+    def get_str_value(self) -> str:
+        stringifier = ConsoleStringifier(c.configuration.to_stringifier_config())
+        return stringifier.create_str_value(self._result)
```

Comment:
Would it be beneficial to also have access to the value and the uncertainties as numeric values, not only as strings? For example,

```py
res = wiz.res(3.14, ...)
# Some further calculation
print(res.value + 1.0)
```

In this case, we should make very clear in the docs that the object returned by `wiz.res()` is non-modifiable, e.g. you cannot set `res.value = 42` afterwards and expect the underlying value to change. It is read-only.

But maybe that is overkill and we should leave it at strings. One could also argue that the user already knows the numerical value upfront, so they could instead just assign it the a local variable before passing it to `wiz.res()` and would be good to go... This would also avoid confusion about the modifiability ob the object returned by `wiz.res()` since when you only get back strings, you probably don't expect that this would modify the underlying value`res.value = "42.0"`. In that case, maybe we can even drop the "str" part of the method names? Since the datatype is clear by the method signature `-> str`.

Reply:
> But maybe that is overkill and we should leave it at strings. 

I agree :-)